packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -534,46 +535,42 @@ class _CupertinoAppState extends State<CupertinoApp> {
   Widget _exitWidgetSelectionButtonBuilder(
     BuildContext context, {
     required VoidCallback onPressed,
+    required String semanticLabel,
     required GlobalKey key,
   }) {
-    return CupertinoButton(
-      key: key,
-      color: _widgetSelectionButtonsBackgroundColor(context),
-      padding: EdgeInsets.zero,
+    return _CupertinoInspectorButton(
       onPressed: onPressed,
-      child: Icon(
-        CupertinoIcons.xmark,
-        size: 28.0,
-        color: _widgetSelectionButtonsForegroundColor(context),
-        semanticLabel: 'Exit Select Widget mode.',
-      ),
+      semanticLabel: semanticLabel,
+      icon: CupertinoIcons.xmark,
+      buttonKey: key,
     );
   }
 
   Widget _moveExitWidgetSelectionButtonBuilder(
     BuildContext context, {
     required VoidCallback onPressed,
+    required String semanticLabel,
     bool isLeftAligned = true,
   }) {
-    return CupertinoButton(
+    return _CupertinoInspectorButton.iconOnly(
       onPressed: onPressed,
-      padding: EdgeInsets.zero,
-      child: Icon(
-        isLeftAligned ? CupertinoIcons.arrow_right : CupertinoIcons.arrow_left,
-        size: 32.0,
-        color: _widgetSelectionButtonsBackgroundColor(context),
-        semanticLabel:
-            'Move "Exit Select Widget mode" button to the ${isLeftAligned ? 'right' : 'left'}.',
-      ),
+      semanticLabel: semanticLabel,
+      icon: isLeftAligned ? CupertinoIcons.arrow_right : CupertinoIcons.arrow_left,
     );
   }
 
-  Color _widgetSelectionButtonsForegroundColor(BuildContext context) {
-    return CupertinoTheme.of(context).primaryContrastingColor;
-  }
-
-  Color _widgetSelectionButtonsBackgroundColor(BuildContext context) {
-    return CupertinoTheme.of(context).primaryColor;
+  Widget _tapBehaviorButtonBuilder(
+    BuildContext context, {
+    required VoidCallback onPressed,
+    required String semanticLabel,
+    required bool defaultTapBehaviorEnabled,
+  }) {
+    return _CupertinoInspectorButton.toggle(
+      onPressed: onPressed,
+      semanticLabel: semanticLabel,
+      icon: CupertinoIcons.cursor_rays,
```

Comment:
nit: maybe add a comment that this same icon is used in the DevTools Inspector, so if this ever changes the icon should be updated there too.

Reply:
Done!

Reply:
done!